tools/c7n_azure/c7n_azure/constants.py

```diff
@@ -25,6 +25,19 @@
 FUNCTION_PACKAGE_SAS_EXPIRY_DAYS = 365 * 10  # 10 years
 FUNCTION_AUTOSCALE_NAME = 'cloud_custodian_default'
 
+"""
+Azure Container Host
+"""
+CONTAINER_EVENT_TRIGGER_MODE = 'container-event'
```

Comment:
This is the mode type string in the policy though - they have to be unique otherwise when you run custodian it will start provisioning Azure functions.  They have their own modes registered.

If you mean to keep them separate but drop the word container, then I think it is worth discussing - originally I called it `selfhost` but all the ways I was going to actually support using it involved running it from docker and container host sounded better.